logger.go

```diff
@@ -4,9 +4,10 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"sync"
 )
 
-var lflags = log.Ldate | log.Lmicroseconds | log.Lshortfile
+var std = &logger{l: DefaultLogger(), lvl: LogLevelInfo}
```

Comment:
I almost want to suggest the default should be Warn or higher, but then given how little metafora logs...I don't care :p